src/bosh-softlayer-cpi/action/delete_disk.go

```diff
@@ -1,7 +1,9 @@
 package action
 
 import (
```

Comment:
It's a good habit to import packages orderly(https://github.com/golang/go/wiki/CodeReviewComments#imports), such as:
```
import (
  standard libraries

  external libraries

  internal libraries
)
```

https://github.com/cloudfoundry/bosh-softlayer-cpi-release/blob/master/src/bosh-softlayer-cpi/action/create_vm.go#L3-L22